config/production/airflow-vpc.yaml

```diff
@@ -0,0 +1,10 @@
+template:
+  type: http
+  url: "{{stack_group_config.aws_infra_templates_root_url}}/v0.3.0/templates/VPC/vpc-20-private.yaml"
+stack_name: airflow-vpc-test
+
+parameters:
+  VpcName: airflow-vpc-test
```

Comment:
When I was considering the VPC name before reserving a CIDR block [here](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/352976898/Infrastructure+Inventory#:~:text=orca%2Dvpc%20(TBD)-,172.24.0.0/20,-VPC%20for%20Airflow), I initially went with `airflow-vpc`, but I didn't want it to be specific to Airflow. I think `orca-vpc` is more flexible and give us some room to grow. 

In any case, could you make sure that the file name, stack name, and VPC name are all consistent? When you, don't forget to update the ID references in `airflow-ec2.yaml`. Thanks!

Reply:
changed in latest commit